boards/arm/frdm_k22f/doc/index.rst

```diff
@@ -0,0 +1,282 @@
+- Easy access to MCU input/output through Arduino* R3 compatible I/O connectors
+- Programmable OpenSDAv2 debug circuit supporting the CMSIS-DAP Interface
+  software that provides:
+
+  - Mass storage device (MSD) flash programming interface
+  - CMSIS-DAP debug interface over a driver-less USB HID connection providing
+    run-control debugging and compatibility with IDE tools
+  - Virtual serial port interface
+  - Open source CMSIS-DAP software project
+
+- Optional SDHC
+
+For more information about the K22F SoC and FRDM-K22F board:
+
+- `K22F Website`_
+- `K22F Datasheet`_
+- `K22F Reference Manual`_
+- `FRDM-K22F Website`_
+- `FRDM-K22F User Guide`_
+- `FRDM-K22F Schematics`_
+
+Supported Features
+==================
+
+The frdm_k22f board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+| ADC       | on-chip    | adc                                 |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | soc flash                           |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | USB device                          |
++-----------+------------+-------------------------------------+
+| SENSOR    | off-chip   | fxos8700 polling;                   |
+|           |            | fxos8700 trigger                    |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
```

Comment:
Let's change this to link to the defconfig file:
```
The default configuration can be found in the defconfig file:
:zephyr_file:`boards/arm/frdm_k22f/frdm_k22f_defconfig`.
```

Reply:
Thats a nice little tag, I'll try to remember that, will do and thanks for reviewing

Reply:
Of course, thanks for looking at this